static_report/src/components/shared/Columns/ColumnDetailMasterList/ColumnDetailListItem.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  Box,
+  Divider,
+  Flex,
+  FlexProps,
+  Icon,
+  Progress,
+  Text,
+} from '@chakra-ui/react';
+import { ColumnSchema } from '../../../../sdlc/single-report-schema';
+import {
+  formatColumnValueWith,
+  formatIntervalMinMax,
+} from '../../../../utils/formatters';
+import {
+  getColumnMetricRatio,
+  getIconForColumnType,
+} from '../../../../utils/transformers';
+import { NO_VALUE } from '../constants';
+
+interface Props {
+  baseColumnDatum?: ColumnSchema;
+  targetColumnDatum?: ColumnSchema;
+  onSelect: (arg: string) => void;
+  isActive: boolean;
+  hasSplitView?: boolean;
+}
+export function ColumnDetailListItem({
+  baseColumnDatum,
+  targetColumnDatum,
+  onSelect,
+  isActive,
+  hasSplitView,
+  ...props
+}: Props & FlexProps) {
+  const { icon, backgroundColor } = getIconForColumnType(baseColumnDatum);
+  const fallbackColumnDatum = baseColumnDatum || targetColumnDatum;
```

Comment:
`targetColumnDatum` first?

Reply:
`base` should always be fallback AFAIK when comparing pairs of tables or columns. 
This is because we are "basing" it on `base` which provides us the directional delta difference (e.g. age `base: 12` -> `target:18`)